requirements/base.txt

```diff
@@ -49,7 +48,6 @@ ipdb==0.11
 django-autotranslate==1.1.0
 svgutils==0.3.0
 Werkzeug[watchdog]==0.14.1
-matplotlib
```

Comment:
why was this removed btw?

Reply:
Ahhh this caused the docker image build to hang on DigitalOcean as well as Google Cloud Platform and AWS. Not sure why. I'll restore this now.